angucomplete-alt.js

```diff
@@ -298,20 +298,35 @@
         }
       }
 
+      function handleSubmitOnEnter(event) {
+        if (event && !scope.submitOnEnter) {
```

Comment:
Testing locally, it seems the `scope.submitOnEnter` value is always either the string `"true"` or `"false"`, instead of a boolean value.